concent_api/core/views.py

```diff
@@ -667,6 +605,21 @@ def handle_send_force_subtask_results_response(request, client_message):
         provider_public_key  = force_subtask_results.last().auth.provider_public_key_bytes,
         requestor_public_key = client_public_key,
     )
+    subtask = update_subtask(
+        subtask_id                  = client_message_subtask_id,
+        requestor_public_key        = client_public_key,
+        state                       = state,
+        task_to_compute             = task_to_compute,
+        report_computed_task        = report_computed_task,
+        subtask_results_accepted    = subtask_results_accepted,
+        subtask_results_rejected    = subtask_results_rejected,
+    )
+    store_pending_message(
+        response_type       = response_type,
+        client_public_key   = subtask.provider.public_key_bytes,
```

Comment:
This should be sent to the provider inside `ForceSubtaskResultsResponse`.

Reply:
:+1: I wasn't aware of this, we should create separate issue for creating test for this scenario.

Reply:
And it sent to the provider inside `ForceSubtaskResultsResponse`

```python
    elif database_message.response_type == PendingResponse.ResponseType.ForceSubtaskResultsResponse.name:  # pylint: disable=no-member
        subtask_results_accepted = deserialize_message(database_message.subtask.subtask_results_accepted.data.tobytes())
        mark_message_as_delivered(database_message)
        response_to_client = message.concents.ForceSubtaskResultsResponse(
            subtask_results_accepted = subtask_results_accepted,
        )
        logging.log_receive_message_from_database(
            response_to_client,
            database_message.client.public_key,
            database_message.response_type,
            database_message.queue
        )
        return response_to_client
```